web-ui/src/main/resources/catalog/locales/cz-editor.json

```diff
@@ -2,6 +2,7 @@
     "add-extent-from-geokeywords": "Z klíčových slov vypočítat rozměr",
     "add-extent-from-geokeywords-help": "Vyhledat všechna klíčová slova z tezauru typových míst a přidat odpovídající rozměry.",
     "addCRS-help": "Přidat nové CRS",
+    "addConcept": "Add concept to keyword list",
```

Comment:
FYI, you don't need to populate all JSON files for translations. Only add to en-*.json then transifex get the new terms and when we update from transifex it will populate other languages with the translation made in transifex or the default english one.